perllib/FixMyStreet/App/Controller/Auth/Social.pm

```diff
@@ -231,6 +244,10 @@ sub oidc_callback: Path('/auth/OIDC') : Args(0) {
             );
             $c->res->redirect($url);
             $c->detach;
+        } elsif ($c->get_param('state') &&$c->get_param('state') eq 'password_change' && $c->user_exists) {
```

Comment:
Missing space (also, as I'm here, any reason this is in a different order to the same if a few lines down?).

Reply:
No good reason, no :)